src/tests/Microsoft.PowerFx.Performance.Tests/UnitTests/PerformanceTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using Xunit;
+
+namespace Microsoft.PowerFx.Performance.Tests.UnitTests
+{
+    public class PerformanceTests
+    {
+        [Fact]
+        public void PerformanceTest()
+        {
+            var pt = new PerformanceTest1();
+            pt.GlobalSetup();
+            pt.N = 20;
+
+            for (var i = 0; i < 10000; i++)
```

Comment:
>for (var i = 0; i < 10000; i++) [](http://example.com/codeflow?start=12&length=31)

why bother looping here?

Reply:
to run VS profiler

Reply:
Sometimes convenient to run VS profiles, hence this test

Reply:
Just get this from env variable. In CI, defaults to 1.  If we want to profile, can set to 10k.